### ISSUE TITLE
fix autoloader causing class to be redeclared

### ISSUE THREAD

**Issue by gnowland** [gnowland/wp-jet-fuel#9]
```
require(file_exists($composer = __DIR__ . '/vendor/autoload.php') ? $composer : __DIR__ . '/dist/autoload.php');
```
Causing "Cannot redeclare class" error

**Comment by gnowland** [comment]
closed by 0641927

**Comment by gnowland** [comment]
thanks to https://dev.to/blocksbylukas/the-magic-behind-autoloading-php-files-using-composer-1e19
